charts/newrelic-infrastructure/templates/secret.yaml

```diff
@@ -1,14 +1,2 @@
-{{- if not (include "newrelic.licenseCustomSecretName" .) }}
-{{- if (include "newrelic.licenseKey" . | required "licenseKey or customSecretName and customSecretLicenseKey must be set" ) }}
-apiVersion: v1
-kind: Secret
-metadata:
-  namespace: {{ .Release.Namespace }}
-  labels:
-    {{- include "newrelic.labels" . | nindent 4 }}
-  name: {{ template "newrelic.fullname" . }}-license
-type: Opaque
-data:
-  licenseKey: {{ include "newrelic.licenseKey" . | b64enc }}
-{{- end }}
-{{- end }}
+{{- /* Common library will take care of creating the secret or not. */}}
+{{- include "common.license.secret" . }}
```

Comment:
Missing newline.

Reply:
I am leaving this for the nit-picking commit.

Reply:
Solved in [#881654c](https://github.com/newrelic/nri-kubernetes/pull/383/commits/881654c7b9e3b6a104b4a9a29652cf7526e4c4f8)